### PR TITLE
Translation of email and tweets to Portuguese

### PR DESCRIPTION
**Pull request by yogmel**
For the issue #14 

Email and Tweets translation for Brazilian - Portuguese.

Some notes:
- There are some changes I had to do to keep the 280 tweet-limit:
  - instead of '**completely**', '100%' was used. This is quite normal in Portuguese, although this is not the exact translation;
  - '**expressly**' translation 'expressivamente' was taken out of some tweets.
- In the email, 'Greetings' could be translated to 'Prezados', but again this is a gendered word (masculine). It was used 'Olá' instead, which is 'Hello'. It could be used a more formal greeting such as Good morning/afternoon/night.

Gendered language notes:
- 'Applicants' could be translated to 'candidato', but to keep the language gender-neutral, 'pessoa candidata' was used (which means 'person who is an applicant').

### CODE REVIEW

**Comment by sagesharp** [comment]
This looks good when I run it through Google translate. @contraexemplo can you take a look at this?

File: creative-works/sample-email-PORTUGUESE.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Olá,
+Inscrições estão abertas agora para o Outreachy, um programa de estágio que apoia diversidade em software livre e de código aberto (open source). Outreachy oferece estágios remotos, com duração de três meses, com bolsa de $5,500 e um auxílio de $500 para viagem. Outreachy expressamente convida a se candidatarem pessoas que são mulheres (tanto cis quanto trans), homens trans, e pessoas genderqueer. Nós também convidamos expressamente pessoas residentes e nativas dos E.U.A. de qualquer gênero, que sejam Negras/Afro-americanas, Hispânicas/latinas, Nativo americanas/Índio americanas, nativo alasquianas, nativo havainas ou ilhéus do Pacífico. Qualquer pessoa que sofram de preconceito ou discriminação na área de tecnologia em seus países também é convidada a aplicar.
```
Comment (contraexemplo) [inline creative-works/sample-email-PORTUGUESE.txt]: While that is a perfect translation of what was originally written, it is sounding a bit weird in Portuguese. My suggestion would be:

```suggestion
As inscrições para o Outreachy, um programa de estágio que apoia a diversidade em projetos de software livre e código aberto, agora estão abertas. Outreachy oferece estágios remotos com duração de três meses, uma bolsa de U$5.500 e um auxílio-financeiro de U$500 para viagens, e expressamente convida a se candidatarem mulheres (tanto cis quanto trans), homens trans, e pessoas genderqueer. Nós também convidamos expressamente pessoas residentes e nativas dos E.U.A. de qualquer gênero, que sejam Negras/Afro-americanas, Hispânicas/latinas, Nativo americanas/Índio americanas, nativo alasquianas, nativo havainas ou ilhéus do Pacífico. Qualquer pessoa que sofram de preconceito ou discriminação na área de tecnologia em seus países também é convidada a se candidatar.
```
Changes:
- Text is now sounding more natural in Portuguese.
- Added U$ to make it clear all stipend is in US dollars.
- Localized 5,500 as in Brazil we use dots, not commas.
- In Portuguese, the repetition of the word "Outreachy" makes the text a bit exhausting to read. Added an "(...) and expressively (...)" to the beginning of the second phrase using it to make it a bit better.
- Deleted the first "pessoas que são (...)" as it seems to be unnecessary.
- "Se aplicar" also sounds a bit unnatural. Unfortunately, using "se candidatar" (repeating the same verb as before) seems to be is the only option to me.

File: creative-works/sample-email-PORTUGUESE.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Estagiários do Outreachy trabalham remotamente com mentores experientes em projetos de software livre e de código aberto. Os estágios são normalmente focados em atividades de programação, mas alguns projetos oferecem estágios em experiência do usuário, design gráfico, documentação, desenvolvimento web, marketing, tradução, entre outros.
```
Comment (contraexemplo) [inline creative-works/sample-email-PORTUGUESE.txt]: This part is almost perfectly translated. I would only change two things:

```suggestion
Estagiários do Outreachy trabalham remotamente com mentores experientes em projetos de software livre e de código aberto. Os estágios são normalmente focados em atividades de programação, mas alguns projetos oferecem estágios relacionados a experiência de usuário, design gráfico, documentação, desenvolvimento web, marketing, tradução, entre outros.
```
- Add "related to" ("relacionados a" in Portuguese).
- Not use "do usuário" but "de usuário".

File: creative-works/sample-email-PORTUGUESE.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Os estágios do Outreachy acontecem de 4 de Dezembro de 2018 até dia 4 de Março de 2019. Mais detalhes e os critérios de elegibilidade podem ser acessados aqui:
```
Comment (contraexemplo) [inline creative-works/sample-email-PORTUGUESE.txt]: This is perfect. I would only delete "até dia", use "a" instead and not capitalize month days to conform to Portuguese grammatical rules:

```suggestion
Os estágios do Outreachy acontecem de 4 de dezembro de 2018 a 4 de março de 2019. Mais detalhes e os critérios de elegibilidade podem ser acessados aqui:
```

File: creative-works/sample-email-PORTUGUESE.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+https://www.outreachy.org/apply/eligibility/
+
+Aplicar para o Outreachy é um pouco diferente do processo de outros programas de estágio. Durante o período de inscrição (de 19 de Setembro até 30 de Outubro), é esperado da pessoa candidata a entrar em contato com um ou mais mentores de projeto, e fazer uma contribuição para o projeto. Organizadores do Outreachy notaram que as pessoas canditadas mais preparadas são aquelas que entram em contato com o seu mentor antecipadamente, fazem várias perguntas, e enviam contribuições continuamente durante o período de inscrição.
```
Comment (contraexemplo) [inline creative-works/sample-email-PORTUGUESE.txt]: Again the problem with the verb "apply".

```suggestion
Candidatar-se ao Outreachy é um pouco diferente do processo de outros programas de estágio. Durante o período de inscrição (de 19 de setembro a 30 de outubro), é esperado da pessoa candidata a entrar em contato com um ou mais mentores de projeto, e fazer uma contribuição. A organização do Outreachy notou que as pessoas canditadas mais fortes são aquelas que entram em contato com seus mentores antecipadamente, fazem várias perguntas, e enviam contribuições continuamente durante o período de inscrição.
```
Changes:
- "Aplicar" to "Candidatar-se".
- To make "organizers" gender neutral, I used "organização" (organization) instead so it wouldn't be exhausting to read "pessoas" this, "pessoas" that.
- "Strongest" became "prepared" in the translation but I think "fortes" is a great equivalent in Portuguese also.
- "o seu mentor" became "seus mentores" to keep it gender neutral too.

File: creative-works/sample-email-PORTUGUESE.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+Por favor, entre em contato caso tenham perguntas sobre o programa. As pessoas organizadoras do Outreachy (Karen Sandler, Sage Sharp, Marina Zhurakhinskaya, Cindy Pallares e Tony Sebro) podem ser contatadas a partir do formulário de contato no https://outreachy.org/contact
```
Comment (contraexemplo) [inline creative-works/sample-email-PORTUGUESE.txt]: :+1: but

```suggestion
Por favor, entre em contato caso tenham dúvidas sobre o programa. As pessoas organizadoras do Outreachy (Karen Sandler, Sage Sharp, Marina Zhurakhinskaya, Cindy Pallares e Tony Sebro) podem ser contatadas a partir do formulário de contato no https://outreachy.org/contact
```
- "dúvidas" is a bit better than "perguntas" in this case.

File: creative-works/sample-email-PORTUGUESE.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+Esperamos que você nos ajude a compartilhar os estágios do Outreachy!
```
Comment (contraexemplo) [inline creative-works/sample-email-PORTUGUESE.txt]: The original says "spread the word". The best equivalent I can find in Portuguese in this case is "divulgar". A more literal translation ("espalhar a palavra") has religious connotations.

```suggestion
Esperamos que você nos ajude a divulgar os estágios do Outreachy!
```

File: creative-works/sample-tweets-PORTUGUESE.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
+Tweet padrão:
+
+Por favor RT:
+Estágios do Outreachy são 100% remotos, com bolsa de $5,500, e duram três meses.
```
Comment (contraexemplo) [inline creative-works/sample-tweets-PORTUGUESE.txt]: Oh, those tweets were fun to review. Here's my two cents (we'll be using all 280 characters in this one):

```suggestion
Estágios do Outreachy são 100% remotos, com bolsa de U$5.500, e duram 3 meses.
```

File: creative-works/sample-tweets-PORTUGUESE.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+Outreachy apoia a diversidade em software livre e de código aberto. Inscrições para a rodada de Dezembro 2018 até Março 2019 do @outreachy estão abertas agora: https://www.outreachy.org/apply/
```
Comment (contraexemplo) [inline creative-works/sample-tweets-PORTUGUESE.txt]:
```suggestion
Outreachy apoia a diversidade em projetos de software livre e de código aberto. Inscrições para a rodada de dezembro/2018 a março/2019 do @outreachy estão abertas: https://www.outreachy.org/apply
```

File: creative-works/sample-tweets-PORTUGUESE.txt [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+Continuação do Tweet:
+
+Outreachy expressamente convida globalmente a se candidatarem pessoas que são mulheres (cis e trans), genderqueer e homens trans. Apliquem para os nossos estágios, que são completamente remotos, pagos (bolsa de $5,500), e duram três meses. https://www.outreachy.org/apply/
```
Comment (contraexemplo) [inline creative-works/sample-tweets-PORTUGUESE.txt]:
```suggestion
Outreachy expressamente convida a se candidatarem mulheres (cis e trans), homens trans e pessoas genderqueer globalmente. Candidatem-se aos nossos estágios, que são completamente remotos, pagos (com bolsa de U$5.500), e duram três meses. https://www.outreachy.org/apply
```

File: creative-works/sample-tweets-PORTUGUESE.txt [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+Outreachy também convida expressamente pessoas residentes e nativas dos E.U.A. de qualquer gênero, que sejam Negras/Afro-americanas, Hispânicas/latinas, Nativo americanas/Índio americanas, nativo alasquianas, nativo havainas ou ilhéus do Pacífico. https://www.outreachy.org/apply/
```
Comment (contraexemplo) [inline creative-works/sample-tweets-PORTUGUESE.txt:11]: :+1:

File: creative-works/sample-tweets-PORTUGUESE.txt [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+
+Outreachy convida pessoas de todo lugar do mundo, de grupos com pouca representatividade ou discriminação na área de tecnologia em seus países. Aplique para os nossos estágios, que são 100% remotos, pagos (bolsa de $5,500) e duram três meses. https://www.outreachy.org/apply/
```
Comment (contraexemplo) [inline creative-works/sample-tweets-PORTUGUESE.txt]:
```suggestion
Outreachy convida pessoas de todo lugar do mundo, de grupos com pouca representatividade ou que sofrem discriminação na área de tecnologia em seus países. Candidate-se aos nossos estágios, que são 100% remotos, pagos (bolsa de U$5.500) e duram três meses. https://www.outreachy.org/apply/
```

**Review by contraexemplo — CHANGES_REQUESTED** [review]
Hey @sagesharp and @yogmel! I reviewed this PR closely and suggested some changes to make the text sound more natural in Brazilian Portuguese. Could you take a look and review my review, @yogmel?  

By the way, thank you so much for dedicating some of your time to translate those files!

**Comment by yogmel** [comment]
Hi @contraexemplo 
Thanks for reviewing this PR :)
I believe I applied and resolved all suggested changes.
Let me know if it looks right.

**Comment by contraexemplo** [comment]
@yogmel It is! Thank you!

@sagesharp All set to go now. :tada:

**Comment by sagesharp** [comment]
Thank you! :heart:

**Pull request merged**